### ISSUE TITLE
Wrong number of args (2) passed to: map/to-map

### ISSUE THREAD

**Issue by AdamFrey** [zcaudate/hara#29]
Similar problem to https://github.com/zcaudate/hara/issues/28

```
(time/parse "2016-12-14" "yyyy-MM-dd")
=> clojure.lang.ArityException: Wrong number of args (2) passed to: map/to-map
```

`to-map` takes 3 arguments
https://github.com/zcaudate/hara/blob/master/src/hara/time/data/map.clj#L11

But the two defmethods at the bottom of this file are only passing 2
https://github.com/zcaudate/hara/blob/master/src/hara/time/data/format/java_text_simpledateformat.clj#L76

**Comment by zcaudate** [comment]
oh dear. that's embarrassing. thanks for picking this up.

**Comment by zcaudate** [comment]
okay... I've made the fix for `to-map` to take 2 arguments:
https://github.com/zcaudate/hara/blob/master/src/hara/time/data/map.clj#L29

and an example of a passing test...
https://github.com/zcaudate/hara/blob/master/test/hara/time/data/format_test.clj#L67-L69

 I just didn't know where to put it. If it's urgent, I can quickly push a fixed version out but if you're not in a rush I'll aim to do it within the week.

**Comment by AdamFrey** [comment]
it's not urgent for me, actually.

**Comment by zcaudate** [comment]
this is fixed in v2.5.0
